### ISSUE TITLE
Create "eliciting beliefs" section

### ISSUE THREAD

**Issue by mgutierrezc** [kmlv/MoneyPolitics#70]
- [x] Lit review for this section
- [x] Code section using lit review

**Comment by mgutierrezc** [comment]
@johnjelliott, an additional page will be created. It'll appear after choosing the respective tax parameter and before the results page called BeliefElicitation. An HTML'll be required for that

Fields for BeliefElicitation page:
```
guessed_ranking
guessed_system
```

We need to show the payoffs from that section in our Results page, so now, the payoff variable will store the total payoff of the experiment (including the one of the belief elicitation section).  Check the following fields to understand the new vars and changes

Fields for Result page (including the belief elicitation results):
```
game_payoff (payoff from interaction)
guessed_ranking_payoff
guessed_system_payoff
belief_elicitation_payoff (guessed_ranking_payoff + guessed_system_payoff)
payoff (game_payoff + belief_elicitation_payoff)
```

**Comment by mgutierrezc** [comment]
merge with my branch to get the code version with the backend for this section

**Comment by johnjelliott** [comment]
Sounds good, I will start working on it

**Comment by mgutierrezc** [comment]
Errors were fixed. The only thing left is to tell the players their game payoff under the Progressivity treatment in `Results.html`. @johnjelliott, could you take care of that?

**Comment by mgutierrezc** [comment]
Also, check my pull req with the fixes pls

**Comment by johnjelliott** [comment]
Sounds good

**Comment by johnjelliott** [comment]
I think this is already complete as the payoffs are not under an "if"
statement in the results.html page

**Comment by mgutierrezc** [comment]
When I tested that section, the game payoff (without the ones from the belief elicitacion section) didn't appear

**Comment by mgutierrezc** [comment]
Before pushing the changes to your repo, accept the pull req and merge your branch with `working_space` as I've finished registered there the main code changes

**Comment by johnjelliott** [comment]
I am not sure where the code is throwing an error for you with respect to the results page and progressivity but mine seems to be working fine:
![image](https://user-images.githubusercontent.com/69054534/96287946-81609180-0f97-11eb-9c3b-34ef01ad7c82.png)

**Comment by johnjelliott** [comment]
I have also added session configs for progressivity and am going to add the high chart to the tax progressivity selection page after i update the readME

**Comment by mgutierrezc** [comment]
1. In the picture,it's not clear the difference of the game payoffs from the belief elicitation ones. The player shouldn't just be told the total payoff, there should be one line with its game payoff, another one with the total belief elicitation one and other with the total payoff

1. I asked you to merge with my branch before as I've already changed the session configs for the progressivity treatment. Those changes have been implemented too on "working space"
